assets/src/edit-story/elements/gif/index.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { PanelTypes } from '../../components/panels';
+import { SHARED_DEFAULT_ATTRIBUTES } from '../shared';
+import { MEDIA_DEFAULT_ATTRIBUTES, MEDIA_PANELS } from '../media';
+export { default as Display } from '../image/display';
+export { default as Edit } from '../image/edit';
+export { default as Frame } from '../image/frame';
+export { default as Output } from './output';
+export { default as TextContent } from '../image/textContent';
+export { default as LayerContent } from '../image/layer';
+export { default as LayerIcon } from '../image/icon';
```

Comment:
Sneaky to just straight-up reuse the components.

I do fear that someone editing image components in the future would be unaware of this reuse, so maybe the reused components should be moved to the "abstract" `media` folder and then referenced in both `image` and `gif` instead?

Reply:
I like this approach. I also originally implemented in `elements/index.js` something like this:

```
import * as textElement from './text';
import * as imageElement from './image';
import * as shapeElement from './shape';
import * as videoElement from './video';
import { Output as GifOutput } from './gif'
const gifElement = { ...imageElement, Output: GifOutput }
```

... which is also a less sneaky/more declarative way of doing this. Whichever you prefer!

Reply:
Morten's suggestion makes sense to me. Makes it clear that these are reused and at the same time allows for easier customization of the gif element if needed.